### ISSUE TITLE
API Specification: /stamps_collected_totals

### ISSUE THREAD

**Issue by MystikHub** [MystikHub/sweng-12#5]
We should come up with some kind of expected output from making a request to /stamps_collected_totals on the back end, to help developing the back end route and the front end chart. Deciding on the format of the expected response early should help both teams get the feature done quickly.

From the client specification:
The web app should scale correctly for stores with numerous locations and numerous
schemes. The dummy store I will be providing for you has 2 location and 2 loyalty
schemes. However I may later provide you with a different dummy store that has a
different amount of locations and schemes, and the code should work the same.

**Comment by MystikHub** [comment]
```
/stamps_collected_totals?month=<unix-timestamp>
{
    "stores": [
        {
            "store_name": "001",
            "data": [
                {
                    "loyalty-scheme": "PAV",
                    "total": 30
                }
            ]
        },
        {
            "store_name": "002",
            "data": [
                {
                    "loyalty-scheme": "PAV"
                    "total": 38
                },
                {
                    "loyalty-scheme": "PAVSW"
                    "total": 42
                }
            ]
        }
    ]
}
```

**Comment by MystikHub** [comment]
Feature is done, needs to be merged from `total-stamp-trend` into `master`

**Comment by MystikHub** [comment]
Closed by #47
